packages/atlas/src/components/_overlays/SendTransferDialogs/WithdrawFundsDialog.tsx

```diff
@@ -84,6 +86,7 @@ export const WithdrawFundsDialog: FC<WithdrawFundsDialogProps> = ({
             proxyCallback(updateStatus)
           ),
         onTxSync: async () => onExitClick(),
```

Comment:
We don't need `onSuccessCallback`. We could actually run `fetchPaymentsData` in `onTxSync`. We do that in several places. Check for instance https://github.com/Joystream/atlas/blob/ypp/packages/atlas/src/views/global/NftPurchaseBottomDrawer/NftPurchaseBottomDrawer.tsx#L197

Reply:
good catch, totally missed that, thx!